docs/integration/push-notification.adoc

```diff
@@ -0,0 +1,128 @@
+= Push Notifications
+:imagesdir: ./../../assets/images/
+:toc:
+:toc-title:
+:toclevels: 4
+
+
+== General Overview
+[IMPORTANT]
+=====
+The current release of the agrirouter push notification service misses the field receiver_id. Therefore, in the CURRENT version, push notifications should only be used by systems, that know the receiver of a message, as there can only be one receipient. This fits for:
+- Farming Software
+- CUs (**ATTENTION**: You will not recognize, that a TaskSet is addressed for a specific machine)
+
+For Telemetry Platforms, there is currently no way to check, which VCU shall receive the message.
+
+An update resolving this will be delivered soon.
+=====
+
+=== Standard behaviour
+The normal behaviour of agrirouter is to put incoming messages from other endpoints to the feed of the receiving endpoint(1).
+
+The app instance can send a request for messages to its endpoint (2), agrirouter will search for messages in the feed(3), so that they are forwarded to the outbox(4), where they can be requested from (REST) or will directly be delivered (MQTT)(5).
+
+Received messages need to be confirmed (6) to delete the message from the feed (7).
+++++
+<p align="center">
+ <img src="./../../assets/images/general/message-request.png"><br>
+<i>Receiving messages through feed</i>
+</p>
+++++
+
+
+
+=== Push Notifications
+With push notifications activated, a copy of each message sent to the feed(1) is directly copied to the outbox(2) and delivered to the AppInstance (directly via MQTT or through polling using HTTP)(3).  Pushed messages need to be confirmed(4) as well as requested messages to be deleted from the feed (5).
+
+++++
+<p align="center">
+ <img src="./../../assets/images/general/message-push.png" ><br>
+<i>Receiving messages with push notifications</i>
+</p>
+++++
+
+
+
+[NOTE]
+====
+Messages in the outbox persist for a short time. If an app instance was not connected to agrirouter for a while, the messages will be deleted from the outbox, however, they persist in the message feed. 
```

Comment:
Updated; however it's better if the developer is "unsure" here, because that forces to do the FeedRequest periodicly which is the general information we want to give with that.